indigo_app/templates/indigo_api/task_detail.html

```diff
@@ -102,6 +102,34 @@ <h5>{{ task.title }}</h5>
     </div>
 
     <div class="col-2">
+      <h6>Assigned to</h6>
+      <ul class="list-unstyled">
+        {% if task.assigned_to %}
+          <li>{% user_profile task.assigned_to %}</li>
+        {% else %}
+          <li><em>Unassigned</em></li>
+        {% endif %}
+      </ul>
+
+      {% if task.potential_assignees %}
+        <form method="POST">
+          {% csrf_token %}
+          {% if task.assigned_to %}
+            <button class="btn btn-outline-danger mb-3" type="submit" data-confirm="Are you sure you want to unassign this task?" formaction="{% url 'unassign_task' place=place.place_code pk=task.pk %}">Unassign</button>
+          {% endif %}
+          <div class="dropdown">
+            <button class="btn btn-outline-secondary mb-3 dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Assign to
+            </button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              {% for potential_assignee in task.potential_assignees %}
+                <button class="dropdown-item mb-1" type="submit" name="user_id" value="{{potential_assignee.id }}" formaction="{% url 'assign_task' place=place.place_code pk=task.pk %}">{% user_profile potential_assignee %}</button>
```

Comment:
'{% user_profile %}` creates a link, rather user `{% user_display %}` which is just the display name.

Also, since the form action is now always the same, it can go onto the form element.